src/communication/xcSession.js

```diff
@@ -45,12 +61,14 @@ define(["../javascriptHelper", "../guid", "./xcWebSocketPublisher", "./xcWebSock
 
         Session.prototype.createPublisher = function () {
             var publisher = new Publisher(this.webSocket, this.configuration, this.privateTopic);
+            this.publishers.push(publisher);
```

Comment:
There's no way to unsubscribe publishers/ subscribers.
The 2 lists (publishers/subscribers) create memory leaks.
Adding a release or dispose publisher/subscriber function could be a solution...